Dockerfile.tmpl

```diff
@@ -155,39 +160,39 @@ RUN pip install spacy && \
 # Install GPU specific packages
 {{ if eq .Accelerator "gpu" }}
 # Install GPU-only packages
-RUN pip install pycuda && \
-    pip install pynvrtc && \
-    pip install pynvml && \
-    pip install nnabla-ext-cuda$CUDA_MAJOR_VERSION$CUDA_MINOR_VERSION && \
+RUN pip install pycuda \
+        pynvrtc \
+        pynvml \
+        nnabla-ext-cuda$CUDA_MAJOR_VERSION$CUDA_MINOR_VERSION && \
     /tmp/clean-layer.sh
 {{ end }}
 
-RUN pip install pysal && \
-    pip install seaborn python-dateutil dask python-igraph && \
-    pip install pyyaml joblib husl geopy mne pyshp && \
-    pip install pandas && \
-    pip install flax && \
+RUN pip install pysal \
+        seaborn python-dateutil dask python-igraph \
+        pyyaml joblib husl geopy mne pyshp \
+        pandas \
+        flax && \
     # Install h2o from source.
     # Use `conda install -c h2oai h2o` once Python 3.7 version is released to conda.
     apt-get install -y default-jre-headless && \
-    pip install -f https://h2o-release.s3.amazonaws.com/h2o/latest_stable_Py.html h2o && \
-    pip install tensorflow-gcs-config==2.6.0 && \
-    pip install tensorflow-addons==0.14.0 && \
-    pip install tensorflow_decision_forests==0.2.0 && \
+    pip install -f https://h2o-release.s3.amazonaws.com/h2o/latest_stable_Py.html h2o \
+        tensorflow-gcs-config==2.6.0 \
+        tensorflow-addons==0.14.0 \
+        tensorflow_decision_forests==0.2.0 && \
     /tmp/clean-layer.sh
 
 RUN apt-get install -y libfreetype6-dev && \
     apt-get install -y libglib2.0-0 libxext6 libsm6 libxrender1 libfontconfig1 --fix-missing && \
     # b/198300835 kornia 4.1.0 is not compatible with our version of numpy.
     pip install gensim==4.0.1 && \
```

Comment:
I think you missed removing the && here.

Reply:
```suggestion
    pip install gensim==4.0.1 \
```

Reply:
good catch, thank you